API/API.mli

```diff
@@ -4669,8 +4669,6 @@ sig
   val constant_has_body : Declarations.constant_body -> bool
   val is_opaque : Declarations.constant_body -> bool
   val eq_recarg : Declarations.recarg -> Declarations.recarg -> bool
-  val body_of_constant :
```

Comment:
Oh well, I guess the answer is that I should bypass the API... however this function is needed for simple things such as printing a constant it seems, so I dunno whether the plugins should access to ir or not.